server/chat-commands/core.ts

```diff
@@ -1089,10 +1117,11 @@ exports.commands = {
 
 	timer(target, room, user) {
 		target = toID(target);
-		if (!room.game || !room.game.timer) {
+		// const game = room.getGame(require('../../.server-dist/room-battle').RoomBattle)
+		if (!room.battle || !room.battle.timer) {
```

Comment:
This was intentionally written to be generic for any game with a similar timer API.

This should be fixed in the `RoomGame` definition, not here.